DigitalLearningSolutions.Web.Tests/ViewModels/Common/SearchablePageViewModelTests.cs

```diff
@@ -19,12 +19,24 @@ public class SearchablePageViewModelTests
         public void Setup()
         {
             details = Builder<CentreCourseDetails>.CreateNew()
-                .With(x => x.Courses = Builder<CourseStatisticsWithAdminFieldResponseCounts>
-                    .CreateListOfSize(15)
-                    .All()
-                    .With(g => g.CustomisationName = "v1")
-                    .With((g, i) => g.ApplicationName = NBuilderAlphabeticalPropertyNamingHelper.IndexToAlphabeticalString(i))
-                    .Build().ToArray())

```

Comment:
Does adding `.OrderBy(stats => stats.ApplicationName)` in here fix things?

Reply:
Sorry, just seen this after the merge. There was a lot inconsistency with the test passing locally and then failing in Jenkins, the list was seemly being generated differently. It seems to be the same issue Alex is having in [790](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/974/files#r814782462)